template.yaml

```diff
@@ -612,6 +612,27 @@ Resources:
       LogGroupName: !Sub /aws/lambda/${AWS::StackName}-audit-message-delimiter
       RetentionInDays: 7
 
+  AuditMessageDelimiterDeadLetterQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub ${AWS::StackName}-audit-message-delimiter-dlq
+      KmsMasterKeyId: '{{resolve:ssm:SqsKmsKeyArn}}'
+      MessageRetentionPeriod: 1209600
```

Comment:
Could you leave a comment for future generations to explain how long this is in sensible units (is it a week or something?)

Reply:
Done.